# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1260224829 28800
#      Mon Dec 07 14:27:09 2009 -0800
# Node ID 85389b4e3491a5937453530fb9ad5eab9c61ac59
# Parent  9d447ebfc961074bdf1c3d5cd58a8bffa3dbc07d
Changelog fix for previous patch

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2009-12-07  Rik <octave@nomad.inbox5.com>
+
+	* interpreter/Makefile.am, faq/Makefile.am, liboctave/Makefile.am: Document
+	use of NOTPARALLEL directive.
+
 2009-12-07  Rik <octave@nomad.inbox5.com>
 
 	* refcard/Makefile.am: Add CLEANFILES and MAINTAINERCLEANFILES objects to
 	properly clean directory.
 
 2009-12-05  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Set dependency of *.texi files on
